src/vs/base/parts/tree/browser/tree.css

```diff
@@ -88,34 +90,34 @@
 }
 
 .vs-dark .monaco-tree .monaco-tree-rows.show-twisties > .monaco-tree-row.has-children > .content:before {
-	background-image: url('collapsed-dark.svg');
+	-webkit-mask-image-image: url('collapsed-dark.svg');
 }
 
 .vs-dark .monaco-tree .monaco-tree-rows.show-twisties > .monaco-tree-row.expanded > .content:before {
-	background-image: url('expanded-dark.svg');
+	-webkit-mask-image-image: url('expanded-dark.svg');
 }
 
 .vs-dark .monaco-tree .monaco-tree-rows > .monaco-tree-row.has-children.loading > .content:before {
-	background-image: url('loading-dark.svg');
+	-webkit-mask-image-image: url('loading-dark.svg');
 }
 
 .hc-black .monaco-tree .monaco-tree-rows.show-twisties > .monaco-tree-row.has-children > .content:before	{
-	background-image: url('collapsed-hc.svg');
+	-webkit-mask-image-image: url('collapsed-hc.svg');
 }
 
 .hc-black .monaco-tree .monaco-tree-rows.show-twisties > .monaco-tree-row.expanded > .content:before {
-	background-image: url('expanded-hc.svg');
+	-webkit-mask-image-image: url('expanded-hc.svg');
 }
 
 .hc-black .monaco-tree .monaco-tree-rows > .monaco-tree-row.has-children.loading > .content:before {
-	background-image: url('loading-hc.svg');
+	-webkit-mask-image-image: url('loading-hc.svg');
 }
 
 .monaco-tree-action.collapse-all {
-	background: url('CollapseAll.svg') center center no-repeat;
+	-webkit-mask-image: url('CollapseAll.svg') center center no-repeat;
```

Comment:
I have a feeling this should be removed. I do not see why the tree offers a default collapse all action. If this is being used by clients, it should come from them. My understanding is that this action is outside the tree anyways?

Reply:
I can remove this if it's not used.